src/components/ClaimReport/ReenderClaimInfo.tsx

```diff
@@ -1,4 +1,4 @@
-import { Chip, Typography, Box, Link } from '@mui/material'
```

Comment:
i know this was there before but can we rename it to 'RenderClaimInfo.tsx' later for file searches its nice to have accurate spelling (I know this was not yours, but can you include in this pr anyway?)